configure.sh

```diff
@@ -38,8 +48,9 @@ export SIMPLESAML_SALT="$(LC_CTYPE=C tr -c -d '0123456789abcdefghijklmnopqrstuvw
 export TORQUEDATA_INSTALL_DIRECTORY=/home/vagrant/torquedata
 export TORQUEDATA_SERVER_PORT=5000
 
+# Two names for the same thing so we reassign it here
```

Comment:
Could you expand this comment a bit?  I didn't understand what there were two names for, and therefore didn't understand the motivation behind the reassignment.

Reply:
Done!  Basically these are used in envsubst and exposed a small number of variables whose names shifted across different ansible scripts (those scripts are totally independent, so that destandardization wasn't an issue in isolation).

Worth eventually tweaking the scripts to use a common name but I figured that would make for a small patch later rather than conflating it with the initial setup of this new process.